### DISCUSSION TITLE
NZBHydra

### DISCUSSION THREAD

**Discussion by teedoff087** [selfhst/icons#20] [New Icon Request]
Official repository: https://github.com/theotherp/nzbhydra2

**Comment by shollyethan** [comment]
Recognizing this request but am still searching for an asset to start with that has decent quality.

**Comment by kusold** [comment]
![](https://raw.githubusercontent.com/theotherp/nzbhydra2/refs/heads/master/core/src/main/resources/static/img/logo-more-margin.png)
[Source](https://github.com/theotherp/nzbhydra2/blob/master/core/src/main/resources/static/img/logo-more-margin.png)

They also include the [logo's PSD](https://github.com/theotherp/nzbhydra2/blob/master/misc/images/logo.psd) if you want the full source.

**Comment by shollyethan** [comment]
Was hoping for something better, but was able to work with the PSD. It has been added to the collection - thanks!
